test/CMakeLists.txt

```diff
@@ -121,12 +121,12 @@ if(SFML_ENABLE_COVERAGE AND OpenCppCoverage_FOUND)
 
     add_custom_command(TARGET runtests
                        COMMENT "Run tests"
-                       POST_BUILD COMMAND "${OpenCppCoverage_BINARY}" ARGS --quiet --export_type cobertura:"${CMAKE_BINARY_DIR}/coverage.out" --cover_children --excluded_modules "${COVERAGE_EXCLUDE}" --sources "${COVERAGE_SRC}" --sources "${COVERAGE_INCLUDE}" -- "${CMAKE_CTEST_COMMAND}" --output-on-failure -C $<CONFIG>
-    )
+                       POST_BUILD COMMAND "${OpenCppCoverage_BINARY}" ARGS --quiet --export_type cobertura:${PROJECT_BINARY_DIR}/coverage.out --cover_children --excluded_modules "${COVERAGE_EXCLUDE}" --sources "${COVERAGE_SRC}" --sources "${COVERAGE_INCLUDE}" -- "${CMAKE_CTEST_COMMAND}" --output-on-failure -C $<CONFIG>
```

Comment:
Aren't quotes needed around `${PROJECT_BINARY_DIR}/coverage.out`?

Reply:
Nervermind, after checking [CMake language manual](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#unquoted-argument), I see that quotes are not needed as long as the referenced variables do not contain an unescaped semicolon.

Reply:
That's right. And it turns out if the path to your CMakeLists.txt contains a semicolon in it, CMake bugs out anyways so there's no way that these lack of quotes can cause any issues. SFML already regularly omits these quotes and it's never been an issue as far as I'm aware. It's just one more subtle way to reduce the amount of code we write.